test/unit/node/util.test.ts

```diff
@@ -0,0 +1,15 @@
+import { getEnvPaths } from "../../../src/node/util"
+
+describe("getEnvPaths", () => {
+  it("should return an object with the data, config and runtime path", () => {
+    const actualPaths = getEnvPaths()
+    const expectedProperties = ["data", "config", "runtime"]
+    expectedProperties.forEach((property) => {
+      // This is a funky way to do it rather than calling hasOwnProperty itself
+      // We do this because of this eslint rule
+      // Link: https://eslint.org/docs/rules/no-prototype-builtins
+      const hasProperty = Object.prototype.hasOwnProperty.call(actualPaths, property)
+      expect(hasProperty).toBe(true)
```

Comment:
Would need to export `Paths` to make this work but one alternative:

```suggestion
      expect(actualPaths[property as keyof Paths]).toBeDefined()
```